recipes/lrez/build.sh

```diff
@@ -1,23 +1,35 @@
 #!/bin/bash
-export HTSLIB_ROOT=${PREFIX}/lib
-rm -rf build
-mkdir build
-cd build
-cmake ..
-make -j2
-cd ..
 
-mkdir -p $PREFIX/bin/
 
+export C_INCLUDE_PATH=${PREFIX}/include
+export CPLUS_INCLUDE_PATH=${PREFIX}/include
+export LD_LIBRARY_PATH=${PREFIX}/lib
+export LIBRARY_PATH=${PREFIX}/lib
+
+export CFLAGS="$CFLAGS -I$PREFIX/include"
+export LDFLAGS="-lz -lm -lc -lboost_iostreams -L$PREFIX/lib"
+export CPATH=${PREFIX}/include
 
-cp build/tools/Compare ${PREFIX}/bin/
-chmod +x $PREFIX/bin/Compare
+export CXXFLAGS="-D_GLIBCXX_USE_CXX11_ABI=0 -I$PREFIX/include"
 
-cp build/tools/BamExtractor ${PREFIX}/bin/
-chmod +x $PREFIX/bin/BamExtractor
+mkdir -p ${PREFIX}/bin
+
+# Install bamtools
+cd bamtools
```

Comment:
FYI, you can just specify bamtools as a dependency. The .a file and headers will then be under ${PREFIX}.

Reply:
I'm using an altered version of bamtools, so I actually need the version which is provided in the tar.gz archive.
Also, thank you very much for fixing all my issues! Greatly appreciated.